mod/01_rating/spec/set/metric_type/researched_spec.rb

```diff
@@ -97,7 +97,7 @@
               with_tag("div", with: { class: "metric-info" }) do
                 with_tag "h5", text: "Metric Type:"
                 with_tag("div", with: { class: "RIGHT-Xmetric_type" })
-                with_tag "h5", text: "Tags:"
+                with_tag "h5", text: "Topics:"
```

Comment:
can you confirm with Vignesh that this is correct?

I was thinking that at some point we wanted the cardtype to be Topics but we often wanted the relationship to be called "Tags"